doc/source/catalog_generation.rst

```diff
@@ -23,67 +31,153 @@ It should also be stressed here that the point and segment photometry source lis
+to compute a minimum rms, with the larger of the two rms values (sigma-clipped rms or minimum rms)
+adopted for further use.  Once viable background and background RMS values are determined, 
+two-dimensional images matching the dimensions of the detection image are constructed.
+Through a configuration setting, a user can specify the sigma-clipped statistics algorithm be
+the chosen method used to compute the background and RMS images, though the special case of 
+identically zero background data will always be evaluated and will supersede the user request when 
+applicable.
+
+The final background determination algorithm, **Background Algorithm 3**, the
+`photutils.background.Background2d <https://photutils.readthedocs.io/en/stable/api/photutils.background.Background2D.html>`_ 
+Astropy tool is *only* invoked if the special case identically zero algorithm has not been applied,
+the user has not requested that only the sigma-clipped statistics algorithm be computed, and the 
+skewness value derived using the sigma-clipped statistics is less than a pre-defined and configurable
+threshold (default value 0.5).
+
+The **Background Algorithm 3** uses
+sigma-clipped statistics to determine background and RMS values across the image, but in
+a localized fashion in contrast to **Background Algorithm 2**. An initial low-resolution
 estimate of the background is performed by computing sigma-clipped median values in 27x27 pixel boxes across
 the image. This low-resolution background image is then median-filtered using a 3x3 pixel sample window to
-correct for local small-scale overestimates and/or underestimates. It should be noted these are configurable values.
-Our catalogs use these values deeming them to be the best for the general situation, but users can tune these values to
-optimize for their own data. To this end, users can adjust parameter values "bkg_box_size" and/or
-"bkg_filter_size" in the <instrument>_<detector>_catalog_generation_all.json files in the following path:
+correct for local small-scale overestimates and/or underestimates.  Both the 27 and 3 pixel
+settings are configurable variables for the user. 
+
+Once a background and RMS image are determined using this final technique, a preliminary 
+background-subtracted image is computed so it can be evaluated for the percentage of negative
+values in the illuminated portion of the image. If the percentage of negative values exceeds a
+configurable and defined threshold (default value 15%), the computation of the background and RMS image 
+from this
+algorithm are discarded.  Instead the background and RMS images computed using **Background Algorithm 2**,
+with the associated updates, are ultimately chosen as the images to use.
+It cannot be emphasized enough that a well-determined background measurement, leading to a good
+threshold definition, is very crucial for proper and successful source identification.
+
```

Comment:
This probably should be it's own sub-sub-section, so people can more easily find this information and even realize that it is not part of the discussion of the algorithms themselves.

Reply:
OK. Done in update 7ce4a81.